src/integrationTest/java/uk/gov/hmcts/reform/pip/data/management/service/ValidationServiceTest.java

```diff
@@ -412,7 +412,10 @@ private static Stream<Arguments> nonStrategicParameters() {
             Arguments.of(ListType.SSCS_NORTH_WEST_DAILY_HEARING_LIST,
                          SSCS_LISTS_JSON_FILE),
             Arguments.of(ListType.SSCS_LONDON_DAILY_HEARING_LIST,
-                         SSCS_LISTS_JSON_FILE)
+                         SSCS_LISTS_JSON_FILE),
```

Comment:
This integration test is fine but you are missing integration tests in PublicationManagementTest and in schemaValidation